internal/plan/plan.go

```diff
@@ -247,6 +254,17 @@ func order(services map[string]*Service, names []string, stop bool) ([]string, e
 	return order, nil
 }
 
+func (l *Layer) validateFields() error {
+	for name, service := range l.Services {
+		if service.Command == "" {
```

Comment:
Do we need to add anything else to this list?

Reply:
No, I don't think so -- I just double-checked, and everything else is optional. "override" was required and already checked (in `CombineLayers`), and command is the only other required field (well, optional in individual layers, required in combined).